apis/i2c_master_slave/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "libtock_i2c_master_slave"
+version = "0.1.0"
+authors = [
+    "Tock Project Developers <tock-dev@googlegroups.com>",
+    "Wilfred Mallawa <wilfred.mallawa@wdc.com>",
+]
+license = "Apache-2.0 OR MIT"
+edition = "2021"
+repository = "https://www.github.com/tock/libtock-rs"
+rust-version.workspace = true
+description = "libtock I2C master-slave driver"
+
+[dependencies]
+libtock_platform = { path = "../../platform" }
+
+[dev-dependencies]
+libtock_unittest = { path = "../../unittest" }
```

Comment:
You depend on `libtock_unittest` but don't have any unit tests. Are you planning to add tests, or leave it as-is?

Reply:
Ah that's left over from copying the boilerplate. I'll remove it.